ui/inspectors/system-section.reel/overview.reel/overview.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @module ui/inspectors/system-section.reel/overview.reel
+ */
+var Component = require("montage/ui/component").Component,
+    Model = require("core/model/model").Model;
+
+/**
+ * @class Overview
+ * @extends Component
+ */
+exports.Overview = Component.specialize(/** @lends Overview# */ {
+
+    systemInfo: {
+        value: null
+    },
+
+    templateDidLoad: {
+        value: function () {
+            var self = this,
+                loadingPromises = [];
+            this.isLoading = true;
+            this.systemInfo = {};
+            loadingPromises.push(
+                this.application.dataService.fetchData(Model.SystemGeneral).then(function(systemGeneral) {
```

Comment:
All calls to dataService should be moved in services (`system-service`, `storage-service`, `peer-service`,...) to avoid calling montage-data specific code from the component.